Pipfile

```diff
@@ -24,6 +23,8 @@ numpy = "*"
 google-cloud-pubsub = "*"
 flask-sqlalchemy = "*"
 flask = "*"
+urllib3 = "*"
```

Comment:
we shouldn't be importing urllib3 it is managed as a subdependency of a number of packages, e.g. requests